css/templates/lune/pages/_landing.css

```diff
@@ -30,12 +30,37 @@
  * Layout & typography
  ******************************************************************************/
 .lune-page-landing {
+  height: 100vh;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
+}
+
+.lune-landing-page-wrapper {
   height: auto;
 }
 
 .lune-landing-section {
   height: var(--lune-landing-page-height);
   min-height: var(--lune-landing-page-min-height);
+  /* Required to get the overflow:scroll on .lune-page-landing working */
+  pointer-events: none;
+}
+
+/* As a consequence of setting pointer-events: none, to make CTA tappable/hoverable */
+.lune-page-landing .ampstart-btn {
+  pointer-events: all;
+}
+
+/*
+ * This pseudo-element is used to space evenly the rest of the content in the page,
+ * in combination with the `justify-between` and `flex-column` classes applied on it container
+ */
+.lune-landing-section-wrapper::before {
```

Comment:
Need to achieve the required design. This pseudo element allows white space on top of the chair (see comments in CSS too)